### PR TITLE
Adding flags and tests to delete workspace

### PR DESCRIPTION
**Pull request by henriquemoraeszup**
Signed-off-by: Henrique Moraes <henrique.moraes@zup.com.br>

### Description

Adding flag support for `rit delete workspace` command. Some of the refactoring done as well in this PR:
* Removed `dirManager` as a dependency, it only checked for a file existence in a single line
* Workspace list no longer lists the default workspace. There is no reason to show it if we will never delete it
* Trying to delete the default workspace issues an error

### How to verify it
Run `rit delete workspace --name=<workspace>` and check it has been deleted successfully

### Changelog

Added flag support for rit delete workspace command

### CODE REVIEW

**Review by lucasdittrichzup — DISMISSED** [review]

**Review by brunasilvazup — APPROVED** [review]

File: pkg/cmd/delete_workspace.go [hunk 1]
```diff
@@ -20,14 +20,18 @@ import (
...
 	"github.com/spf13/cobra"
 
 	"github.com/ZupIT/ritchie-cli/pkg/formula"
 	"github.com/ZupIT/ritchie-cli/pkg/formula/repo/repoutil"
 	"github.com/ZupIT/ritchie-cli/pkg/prompt"
-	"github.com/ZupIT/ritchie-cli/pkg/stream"
+)
+
+const (
+	nameFlagName = "name"
```
Comment (lucasdittrichzup) [inline pkg/cmd/delete_workspace.go:34]: this variable is conflicting with that of **add_repo**, as they are `const that have the **same name**

Comment (henriquemoraeszup) [reply pkg/cmd/delete_workspace.go:34]: I know, once we merge the add repo branch I will fix it

File: pkg/cmd/add_repo_test.go [hunk 2]
```diff
@@ -45,6 +46,9 @@ func TestAddRepoCmd(t *testing.T) {
 	repoProviders.Add("GitLab", formula.Git{Repos: gitRepositoryWithoutTagsMock, NewRepoInfo: github.NewRepoInfo})
 	repoProviders.Add("Bitbucket", formula.Git{Repos: gitRepositoryErrorsMock, NewRepoInfo: github.NewRepoInfo})
 
+	repoList := repoProviders.List()
+	sort.Strings(repoList)
+
```
Comment (brunasilvazup) [inline pkg/cmd/add_repo_test.go:51]: File changes will not be necessary, as the sort is now in the method list above.

**Review by lucasdittrichzup — APPROVED** [review]

**Pull request merged**